From oliver@neukum.org Mon Feb  4 07:35:58 2008
From: Oliver Neukum <oliver@neukum.org>
Date: Mon, 4 Feb 2008 16:34:11 +0100
Subject: USB: quirks for known quirky audio devices
To: Greg KH <greg@kroah.com>, linux-usb@vger.kernel.org
Message-ID: <200802041634.11921.oliver@neukum.org>
Content-Disposition: inline


RESET_RESUME entries for some sound devices that need it.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/core/quirks.c |    9 +++++++++
 1 file changed, 9 insertions(+)

--- a/drivers/usb/core/quirks.c
+++ b/drivers/usb/core/quirks.c
@@ -33,6 +33,15 @@ static const struct usb_device_id usb_qu
 	/* HP 5300/5370C scanner */
 	{ USB_DEVICE(0x03f0, 0x0701), .driver_info = USB_QUIRK_STRING_FETCH_255 },
 
+	/* Creative SB Audigy 2 NX */
+	{ USB_DEVICE(0x041e, 0x3020), .driver_info = USB_QUIRK_RESET_RESUME },
+
+	/* Roland SC-8820 */
+	{ USB_DEVICE(0x0582, 0x0007), .driver_info = USB_QUIRK_RESET_RESUME },
+
+	/* Edirol SD-20 */
+	{ USB_DEVICE(0x0582, 0x0027), .driver_info = USB_QUIRK_RESET_RESUME },
+
 	/* INTEL VALUE SSD */
 	{ USB_DEVICE(0x8086, 0xf1a5), .driver_info = USB_QUIRK_RESET_RESUME },
 
